lab01-activity.Rmd

```diff
@@ -273,7 +287,7 @@ At the end of every lab, replace TODO with DONE to the left of each task to indi
 |status|task|
 |-|-|
 |DONE|This file has been knit|
-|TODO| My .bashrc file exists|
-|TODO| Miniconda is installed|
-|TODO| I was able to install both Bioconda and samtools using conda|
+|DONE| My .bashrc file exists|
+|DONE| Miniconda is installed|
+|DONE| I was able to install both Bioconda and samtools using conda|
 |TODO| The last few code chunks that run Python code run without errors when I knit this document|
```

Comment:
I ran your python code, but there was not any error. Error is for line 226, as I told you, eval should be changed to FALSE for knitting the file.